src/Math/add.js

```diff
@@ -0,0 +1,6 @@
+let _add = (x,y) => {
```

Comment:
ES6, use `let` or `const` instead of `var`. Err on the side of `const`, as when defining functions. The only time I'll use `let` is when later in my code, I assign the variable a new value, ala
```js
let temp = 4
temp = 5
```

Also, the Arrow function `=>` has some syntactic sugar to allow saving of space. So your whole function can be written as:
`const add = ( x, y ) => x + y`
This is called an implicit return: since the function has a single line, it automatically returns that single line.